docs/user-guide/ignore-code.md

```diff
@@ -62,9 +62,6 @@ Stylelint supports complex, overlapping disabling & enabling patterns:
 /* stylelint-enable foo */
 ```
 
-> [!WARNING]
```

Comment:
```
> [!WARNING]
> Invalid CSS comments—i.e. starting with `//`—are ignored if they are within a selector or value list.
```

Reply:
I am not sure that we need to warn about things that do not work when those are things we do not officially support.

But I don't have a strong opinion about this, happy to make changes here.

Reply:
I am sure my proposed wording sucks. Nonetheless we should at least have a `!NOTE`.

Reply:
The wording is fine :)

I was mainly thinking that we already state elsewhere that our focus is on supporting standard CSS only. Mentioning each aspect/feature that doesn't work for non-standard CSS might not be that needed as we already mention it in general?

Reply:
Maybe mentioning the non-standard support would be warranted then?
i.e. "Invalid CSS comments—i.e. starting with `//`—are supported when found outside of selectors and/or value lists"

Reply:
Maybe something like this:

> Non-standard syntax comments (i.e. `//`) are only minimally supported. They will only work when found outside of selectors or value lists.


----

I prefer `non-standard` over `invalid` in this context.

`//` is widely known syntax for comments in `less` and `scss`.

While most developers won't know that even standard CSS has quirks that make it swallow the next construct after `//`. Those quirks do not make `//` a comment since no comment token or node will be produced by standard CSS tokenizers or parsers.

(the `no-invalid-double-slash-comments` rule has a different context, requiring different terms)